bgpd/bgp_route.c

```diff
@@ -5694,8 +5694,8 @@ int bgp_nlri_parse_ip(struct peer *peer, struct attr *attr,
 		}
 
 		/* Defensive coding, double-check the psize fits in a struct
-		 * prefix */
-		if (psize > (ssize_t)sizeof(p.u)) {
+		 * prefix for the v4 and v6 afi's and unicast/multicast */
+		if (psize > (ssize_t)sizeof(p.u.val)) {
```

Comment:
Seems we have to use the same for flog_err(), right?

Reply:
yep fixed